admin/common_auth/views.py

```diff
@@ -48,6 +57,74 @@ def get_success_url(self):
             redirect_to = reverse('home')
         return redirect_to
 
+class ShibLoginView(RedirectView):
+    form_class = LoginForm
```

Comment:
削除してください。

Reply:
61～65行目を削除します。

```suggestion
class ShibLoginView(RedirectView):
```
上記は
class の指定自体が不要ということでしょうか？